### ISSUE TITLE
Performance difference

### ISSUE THREAD

**Issue by cnnlstm** [leeruibin/SPDInv#1]
Thanks for your great work! 

I tried to use your code to edit images on the PIE-Bench_v1 dataset,  I found the reconstruction quality is much higher than that reported in the paper, but the edit performance is lower.

By the way, I use the **run_SPDInv_P2P.py** code to edit the Cat, but it is very different from your paper.  Are there any environmental errors?

![1_P2P_000000000004](https://github.com/leeruibin/SPDInv/assets/22098071/4fca59be-5cc4-4df9-b5db-099014e4d040)

**Comment by leeruibin** [comment]
Thanks for your attention.

However, The code seems to work fine with me. For this image, you can try to decrease the learning rate or reduce the ddim_step to 20 for better editing output.
![1715226847938](https://github.com/leeruibin/SPDInv/assets/26408630/009732b9-8c86-4055-bb52-859378c4b420)

**Comment by cnnlstm** [comment]
Thanks for your reply. Is the performance in Tab.1 based on the specific parameters of each image?

**Comment by leeruibin** [comment]
We have updated the code and some typos, the performace in Tab. 1 is based the hyper parameters mentioned in our paper.

**Comment by cnnlstm** [comment]
Thanks for your reply, based on the updated codes, I still cannot obtain the same results reported in the paper.

**Comment by leeruibin** [comment]
Note that the performance may be different due to the compression of jpg and the effects of random states in pytorch.
